tap_kafka/serialization/protobuf.py

```diff
@@ -24,11 +24,14 @@ def __call__(self, value, ctx):
                              preserving_proto_field_name=True,
                              including_default_value_fields=True)
 
+def protoc_topic_name(topic: str) -> str:
```

Comment:
Could you please find a better name for the method? 
This name does not describe what it is doing exactly.